utils/auth-helper.ts

```diff
@@ -38,7 +38,7 @@ export function getKeycloakRoles (): Array<string> {
   throw new Error('Error getting Keycloak roles')
 }
 
-// get product subscription authorizations
+/** Gets product subscription authorizations from Auth API. */
```

Comment:
just a comment change?

Reply:
Yes, but this:
- makes the comment into a JSDoc header
- make this method header comment the same as the other methods in this file

This is minor tech debt.

Reply:
Sample JSDoc hint for this method:

![Clipboard - January 7, 2022 10_06 AM](https://user-images.githubusercontent.com/10002889/148590737-1ecbac8c-fbc7-444e-9ede-ee68f7fe8338.png)

Reply:
As discussed in RC, Thor things this change is "sweet", so no PR change needed.